src/endpoint.cpp

```diff
@@ -414,6 +416,10 @@ void Endpoint::_add_sys_comp_id(uint8_t sysid, uint8_t compid)
     if (has_sys_comp_id(sys_comp_id)) {
         return;
     }
+    if ((sniffer_sysid != 0) && ((sys_comp_id >> 8) == sniffer_sysid))
+        log_info("Sniffer sysid %u identified. [%d] is now sniffing all messages",
+                 sniffer_sysid,
+                 fd);
```

Comment:
not sure the _add_sys_comp_id() would be the most appropriate place for this message, but we can move it later.

Reply:
1. Did the squash. 
2. Fixed the braces. BTW, first time I am using clang. Installation that works with the build on Ubuntu 20 not trivial (you need version 13+ and two sym links...). Would you like me to add some words in the Readme in a different pull request?
3. The reason I chose this place for the message: In order that endpoint become a sniffer you just have to add the sniffer id to it. No other action is need. So, once we do that I notified the user so he's aware.

Reply:
Sounds good to me. Thanks for working on this feature

Reply:
My pleasure